NewsCrawlerTrigger/linkCrawler.js

```diff
@@ -0,0 +1,38 @@
+const puppeteer = require("puppeteer");
+const { newsPortalLink } = require("../constants/portal");
+const { KANTIPUR, SETOPATI, RATOPATI, DAINIK_KHABAR } = newsPortalLink;
+
+const scrapeNewsLink = link => {
+  switch (link) {
+    case KANTIPUR:
+      return scrapeKantipurNewsLink();
+    default:
+      return null;
+  }
+};
+
+const scrapeKantipurNewsLink = async () => {
+  try {
+    const browser = await puppeteer.launch();
+    const page = await browser.newPage();
+    await page.goto(KANTIPUR);
+
+    const scrapedData = await page.evaluate(() =>
+      Array.from(document.querySelectorAll("article > h1, h2 > a "))
```

Comment:
Is there a way to store `css selectors` for links in each news website in a json file so that, we can make `scrapeKantipurNewsLink` more generic to be usable for every news site?

Reply:
we can do that but simply it seems different for each site

Reply:
@shrijan-siris I mean we will store css-selectors for each news site in different json file

Reply:
that sounds good